### PR TITLE
Switch to `wrangler deploy` because `wrangler publish` is deprecated.

### PR DESCRIPTION
**Pull request by obi1kenobi**
CI said so:
image

### CODE REVIEW

File: test-exports-cf/src/index.ts [hunk 1]
```diff
@@ -37,16 +37,16 @@ export default {
     ctx: ExecutionContext
   ): Promise<Response> {
 
-    const constructorParameters
```
Comment (nfcampos) [inline test-exports-cf/src/index.ts:40]: Why is this changing?

**Review by nfcampos — COMMENTED** [review]
Seems good minus 1 comment

File: test-exports-cf/src/index.ts (hunk 1 above)
Comment (obi1kenobi) [reply test-exports-cf/src/index.ts:40]: Ugh, sorry about that. I think my editor applied some Prettier rules I had configured for another repo.

I'll just disable that plugin and re-create this change.

File: test-exports-cf/src/index.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default {
         azureOpenAIApiInstanceName: env.AZURE_OPENAI_API_INSTANCE_NAME,
         azureOpenAIApiDeploymentName: env.AZURE_OPENAI_API_DEPLOYMENT_NAME,
         azureOpenAIApiVersion: env.AZURE_OPENAI_API_VERSION,
-      }
```
Comment (obi1kenobi) [inline test-exports-cf/src/index.ts:46]: This is just some trailing whitespace that my editor removed, I can leave it be if you'd prefer — all the same to me.

**Pull request merged**